indigo_app/static/javascript/indigo/views/library.js

```diff
@@ -165,11 +179,14 @@
         docs[work.get('id')] = work.documents().models;
       });
 
-      if (filters.status !== 'all') {
+      if (filters.status) {
+        var draft = _.contains(filters.status, 'draft'),
+            published = _.contains(filters.status, 'published');
+
         // filter documents by status
         works = filterWorksByDocs(function(doc) {
-          return (filters.status === "draft" && doc.get('draft') ||
-                  filters.status === "published" && !doc.get('draft'));
+          return (draft && doc.get('draft') ||
+                  published && !doc.get('draft'));
```

Comment:
@longhotsummer This block is taking `works` from 2 stubs down to 0 when `Only stubs` is selected, which stands to reason because the stubs don't have any docs to filter by (if I've understood correctly). When `Exclude stubs` is selected it takes `works` from 11 down to 4, because I have 7 works that aren't explicitly marked as stubs but don't have documents.

Reply:
Oh. 🤕

Reply:
Fixed.